WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -49,7 +52,7 @@ public static void Show(object viewModel)
 		NotificationManager?.Show(viewModel);
 	}
 
-	private static bool TryGetNotificationInputs(ProcessedResult result, [NotNullWhen(true)] out string? message)
+	private static bool TryGetNotificationInputs(decimal fiatExchangeRate, ProcessedResult result, [NotNullWhen(true)] out string? message)
```

Comment:
nit, but swap the order of parameters as `result` is more important than `fiatExchangeRate`.